release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -492,6 +492,14 @@ With a smaller number of clusters, the reduction in installation time is less si
 
 For more information, see xref:../edge_computing/ztp-deploying-far-edge-sites.adoc#ztp-sno-accelerated-ztp_ztp-deploying-far-edge-sites[Accelerated provisioning of {ztp}].
 
+[id="ocp-4-16-edge-computing-ipsec-encryption-for-managed-clusters-ztp_{context}"]
+==== Deploying IPsec encryption to managed clusters with {ztp} and {rh-rhacm} (Technology Preview)
+
+You can now enable IPsec encryption in managed {sno} clusters that you deploy with {ztp} and {rh-rhacm-first}.
+You can encrypt both internal pod-to-pod cluster traffic between nodes and external traffic between pods and IPsec endpoints external to the managed cluster.
+All pod-to-pod network traffic between nodes on the OVN-Kubernetes cluster network is encrypted with IPsec in Transport mode.
+
+For more information, see xref:../edge_computing/ztp-deploying-far-edge-sites.adoc#ztp-configuring-ipsec-using-ztp-and-siteconfig_ztp-deploying-far-edge-sites[Configuring IPsec encryption for {sno} clusters using {ztp} and SiteConfig resources].
```

Comment:
I assume this module is not in place yet? As the link goes to the top of the assembly.

Reply:
Yes - needs docs merged too.